app/server/lib/sendAppPage.ts

```diff
@@ -24,9 +25,14 @@ import * as fse from 'fs-extra';
 import * as handlebars from 'handlebars';
 import jsesc from 'jsesc';
 import * as path from 'path';
-import difference = require('lodash/difference');
+import { difference, trimEnd } from 'lodash';
 
-const translate = (req: express.Request, key: string, args?: any) => req.t(`sendAppPage.${key}`, args);
+const { escapeExpression } = handlebars.Utils;
+
+const translateEscaped = (req: express.Request, key: string, args?: any) => {
```

Comment:
Any reason not to write it as just
```
function translateEscaped(req, key, args) {
...
}
```

It would be helpful to add a docstring with the motivation for this method and the problem that it solves.

Tiny thing: often in Grist codebase the pair of variables req and res are used together as express.Request and express.Response. Maybe use a different name than res here?